src/hivpy/common.py

```diff
@@ -35,6 +35,8 @@ class SexType(IntEnum):
     Male = 0
     Female = 1
 
+def opposite_sex(sex: SexType):
+    return (1 - sex)
```

Comment:
Personally I would prefer to keep the abstraction here, so something like
```python
return SexType.Male if sex is SexType.Female else SexType.Female
```

Reply:
Ah, but that doesn't work if called on a whole column, does it? Hm. There's probably a way to vectorise this while preserving the abstraction but I guess it's not crucial.